modules/buttons/info.html

```diff
@@ -0,0 +1,33 @@
+<style>
+    .info {background-color: #2196F3;} /* Blue */
+    /*.info:hover {background: #0b7dda;}*/
+    button{
+        color: white; 
+        border-radius: 4px;
+        text-align: center;
+        border: none;
+        padding: 14px 28px;
+    }
+    .icon,.text {
+        vertical-align: middle;
+        display: inline-block;
+        align-items: center;
+    }
+    
+</style>
+
+<body>
```

Comment:
Do we really need the body tag?

Reply:
Not required, I was using vscode and body was required to view the live changes, anyways I can remove that too.